README.md

```diff
@@ -253,17 +253,44 @@ If you want to restrict navigation between your slides (i.e. the user has to tog
 All you have to do is implement `ISlidePolicy` in your slides:
 ```java
 public final class MySlide extends Fragment implements ISlidePolicy {
+/* Usage Example part 1 (Create a public variable for the CheckBox):
+ * private CheckBox checkBox;
+ */
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.myslide_layout, container, false);
+	
+	/* Usage Example part 2 (Assign the CheckBox view to the variable) :
+	 * checkBox = view.findViewById(R.id.checkboxid);
+	 */
+	
+	
+        return view;
+    }
     @Override
     public boolean isPolicyRespected() {
-        return // If user should be allowed to leave this slide
+    /* Example part 3 (Returns true if the user checked the checkbox and false if he didn't):
+     * return checkBox.isChecked();
+     */
+        return // You should return true if policy is respected or false if it isn't.
```

Comment:
Please keep only this line and remove the others.

Reply:
This line is meant to complete the others. To make a full step by step documentation.